docs/source/managed-federation/setup.mdx

```diff
@@ -36,35 +71,49 @@ const gateway = new ApolloGateway({
+</ExpansionPanel>
 
-<ObtainGraphApiKey />
+If you've already set up Apollo Federation _without_ connecting your router to GraphOS, you're probably passing the `--supergraph` (or `-s`) option to the Apollo Router's startup command:
+
+```sh
+./router --config ./router.yaml --supergraph ./your-local-supergraph.graphql
+```
 
-After obtaining your graph API key, you set two environment variables in your gateway's environment. If you're using a `.env` file with a library like [`dotenv`](https://www.npmjs.com/package/dotenv), those environment variables look like this:
+The `--supergraph` option is specific to _non_-managed federation, in which supergraph schema composition is performed via the Rover CLI and provided via a file path.
 
-```sh title=".env"
-APOLLO_KEY=<YOUR_GRAPH_API_KEY>
-APOLLO_GRAPH_REF=<YOUR_GRAPH_ID>@<VARIANT>
+_Remove_ the `--supergraph` option (but leave `--config` if it's present):
+
+```sh
+./router --config ./router.yaml
 ```
 
-You can also set this value directly in the command you use to start your gateway.
+> With managed federation, composition is instead performed by _GraphOS_, and the router regularly polls for an updated schema. This enables you to add, remove, and modify your subgraphs _without_ needing to restart your router.
 
-The `APOLLO_GRAPH_REF` environment variable tells the gateway which variant of which graph to use (for example, `my-graph-id@production`). You can find your variant's graph ref at the very top of its README page in Studio.
 
-> When running your gateway in an environment where outbound traffic to the internet is restricted, consult the [directions for configuring a proxy](/apollo-server/security/proxy-configuration) within Apollo Server.
+## 4. Connect your router to GraphOS
+
+Like your subgraphs, your router uses a graph API key to identify itself to GraphOS.
+
+<ObtainGraphApiKey />
+
+After obtaining your graph API key, you set the following environment variables in your router's environment:
+
+- `APOLLO_KEY` (set this to your graph API key)
+- `APOLLO_GRAPH_REF`
+    - This variable tells the router which variant of which graph to use (for example, `docs-example-graph@production`). You can find your variant's graph ref at the very top of its README page in GraphOS Studio.
 
-## 5. Deploy the modified gateway
+## 5. Deploy the modified router
 
-You can now deploy your modified gateway to begin fetching your federated schema from Studio instead of directly from your subgraphs.
+You can now deploy your modified router to begin fetching your supergraph schema from GraphOS instead of using a locally generated one.
 
-On startup, your gateway will use its API key to fetch its federation config from Apollo. It can then begin executing operations across your subgraphs.
+On startup, your router will use its API key to fetch its federation config from GraphOS. It can then begin executing operations across your subgraphs.
```

Comment:
I think "federation config" is an ancient reference to the original gateway's method for constructing a supergraph, I think we can get rid of it 😄

```suggestion
On startup, your router will use its API key to fetch its supergraph from GraphOS. It can then begin executing operations across your subgraphs.
```